pkg/services/sqlstore/migrations/preferences_mig.go

```diff
@@ -15,6 +15,7 @@ func addPreferencesMigrations(mg *Migrator) {
 			{Name: "home_dashboard_id", Type: DB_BigInt, Nullable: false},
 			{Name: "timezone", Type: DB_NVarchar, Length: 50, Nullable: false},
 			{Name: "theme", Type: DB_NVarchar, Length: 20, Nullable: false},
+			{Name: "nav_position", Type: DB_NVarchar, Length: 20, Nullable: false},
```

Comment:
Aha, I see what you did here, the migrations are unique so you can't add the new column to existing migrations. Instead you need to create a new migration. 

I think the label `Update preferences table charset` or `Add column team_id in preferences` is what makes the migration unique and are checked by the migration system and there is a `migration_log` table with all the migrations in the DB.

Reply:
And this new migration should use the feature toggle of course 😄